### ISSUE TITLE
Send emails to users after assignment

### ISSUE THREAD

**Issue by RaduCStefanescu** [code4romania/expert-consultation-api#122]
When we assign users to the documents, we need to notify them via email.

**Comment by trestianp** [comment]
I can start working on this but it depends on #121 . @RaduCStefanescu  Should I include this change in that Pull Request or wait for that one to be merged then create a separate PR for this task ?

**Comment by RaduCStefanescu** [comment]
Fixed by #133
